zebra-script/src/lib.rs

```diff
@@ -42,6 +42,10 @@ pub enum Error {
     TxCoinbase,
     /// unknown error from zcash_script: {0}
     Unknown(zcash_script_error_t),
+    /// transaction is invalid according to zebra_chain (not a zcash_script error)
+    //TODO: ideally this should be `#[from] zebra_chain::Error` but that does
+    //not implement Clone
+    TxInvalid,
 }
```

Comment:
```suggestion
    TxInvalid(#[from] std::sync::Arc<zebra_chain::Error>),
}

impl From<zebra_chain::Error> for Error {
    fn from(value: zebra_chain::Error) -> Self {
        std::sync::Arc::new(value).into()
    }
}
```

Reply:
I think it was small enough so I sneaked this in b69d1a5. Let me know if you'd prefer a separate PR

Reply:
I'm happy for it to go in this PR. 

Could we implement `PartialEq` for the `zebra_script` Error type instead?